cpp/modmesh/toggle/profileRadixTree.hpp

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+/*
+ * Copyright (c) 2019, Yung-Yu Chen <yyc@solvcon.net>
```

Comment:
I am wondering if this is expected, or simply a typo of copy-paste.

Reply:
I believe @q40603 created this for-discussion PR from a local WIP branch.  We do not need to pay attention to formatting for a PR that is not supposed to be merged.